module/applications/actor/character-sheet.mjs

```diff
@@ -148,7 +148,7 @@ export default class ActorSheet5eCharacter extends ActorSheet5e {
         hasActions: false, dataset: {type: "class"}, isClass: true },
       active: {
         label: "DND5E.FeatureActive", items: [],
-        hasActions: true, dataset: {type: "feat", "activation.type": "action"} },
+        hasActions: true, dataset: {type: "feat", "system.activation.type": "action"} },
```

Comment:
If we're changing these datasets, the PR/issue should probably be marked as breaking.